tests/cpptests/test_cpp_llapi.cpp

```diff
@@ -820,3 +820,35 @@ TEST_F(LLApiTest, testScorer) {
   RediSearch_ResultsIteratorFree(it);
   RediSearch_DropIndex(index);
 }
+
+TEST_F(LLApiTest, testStopwords) {
+  // Check default stopword list
+  RSIndex* index = RediSearch_CreateIndex("index", NULL);
+  ASSERT_EQ(RediSearch_StopwordsList_Contains(index, "is", strlen("is")), 1);
+  ASSERT_EQ(RediSearch_StopwordsList_Contains(index, "Redis", strlen("Redis")), 0);
+  // check creation of token node
+  RSQNode *node = RediSearch_CreateTokenNode(index, "doesnt_matter", "is");
+  ASSERT_EQ((size_t)node, 0);
+  node = RediSearch_CreateTokenNode(index, "doesnt_matter", "Redis");
+  ASSERT_NE((size_t)node, 0);
+  RediSearch_QueryNodeFree(node);
+  RediSearch_DropIndex(index);
+
+  // Check custom stopword list
+  const char *words[] = {"Redis", "Labs"};
+  RSIdxOptions opts1 = { .stopwords = words, .stopwordsLen = 2 };
```

Comment:
An index option is an opaque object, needs to be created with `RediSearch_CreateIndexOptions`. I believe it's a good idea that the tests will be using the LLAPI the same way the user will.

Reply:
You are completely right! :+1: